.github/workflows/ci.yml

```
@@ -1,74 +1,100 @@
+
+      - name: Install dependencies
+        run: yarn install --immutable
+
       - name: Lint
         run: yarn run lint
+
       - name: Test
         run: yarn run test
+
       - name: Codecov
         uses: codecov/codecov-action@v3
```

actionlint (1.7.4)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)